src/neuroconv/datainterfaces/ecephys/openephys/openephysroutingdatainterface.py

```diff
@@ -0,0 +1,57 @@
+from abc import ABC
+from pathlib import Path
+from typing import Optional
+
+from .openephysdatainterface import OpenEphysBinaryRecordingInterface
+from .openephyslegacydatainterface import OpenEphysLegacyRecordingInterface
+from ....utils import FolderPathType
+
+
+class OpenEphysRecordingInterface(ABC):
```

Comment:
Does this actually have to be an abstract class? What do we gain by doing that (since not using any abstract methods?)

Reply:
No, I based it on the EXTRACT extractor that redirects. I changed it to inherit from `BaseRecordingExtractorInterface`, is that correct?

Reply:
Hmm I can't recall why we did that back then either hehe

Technically, with proper routing in `__new__`, it wouldn't produce any different behavior either way

It just also wouldn't make use of any the `ABC` behavior either

Was just curious if there was something I was overlooking